components/TaskCard.vue

```diff
@@ -0,0 +1,49 @@
+<template>
+    <div class="card">
+      <div class="cardDescription">{{ cardData.description }}</div>
+      <div style="margin-top:9px; margin-bottom:26px;" class="caption">{{ cardData.createdAt }}</div>
+      <div style="display:flex; gap:8px;">
```

Comment:
Don't use a style like this. Add these properties in `<style>` section or create classes.

Reply:
Resolved